aspnet/mvc/views/forms/sample/final/ViewModels/CountryViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNet.Mvc.Rendering;
+using System.Collections.Generic;
+
+public class CountryViewModel
+{
+    public string Country { get; set; }
+
+    public List<SelectListItem> Countries { get; private set; }
+
+    public CountryViewModel()
```

Comment:
Much easier to understand if the constructor is placed early in the class, after fields but not properties. This and similar constructors in other model classes looks too much like a method.